catalyst_voices_packages/catalyst_cardano_serialization/lib/src/builders/transaction_builder.dart

```diff
@@ -38,13 +38,34 @@ final class TransactionBuilder extends Equatable {
   /// The transaction metadata as a list of key-value pairs (a map).
   final AuxiliaryData? auxiliaryData;
 
-  /// The list of public key hashes of addresses
-  /// that are required to sign the transaction.
+  /// Validity interval start as integer.
+  final SlotBigNum? validityStart;
+
+  /// Mint as a non-zero uint64 multiasset.
+  final MultiAsset? mint;
+
+  /// The transaction metadata as a list of key-value pairs (a map).
+  final ScriptData? scriptData;
+
+  /// Collateral inputs as nonempty set.
+  final Set<TransactionInput>? collateralInputs;
+
+  /// The list of public key hashes of addresses that are required to sign the
+  /// transaction. A nonempty set of addr keyhashes.
```

Comment:
cspell throws an error on `keyhashes`, can we change it to:

```suggestion
  /// transaction. A nonempty set of addr key hashes.
```